string action_name                     # action to be taken
string model_name                 # name of the model to take action
